README.md

```diff
@@ -51,30 +53,49 @@ npm install @vgs/collect-js
 
 ## How to use
 
-The imported function inserts the `<script>` tag to the document head or body and returns the `VGSCollect` instance as the result of resolved Promise. The script won't be loaded until `loadVGSCollect()` invoked. In order to speed up cross-domain loading, `dns-prefetch` and `preconnect` were added as a side effect.
+The imported function inserts the `<script>` tag to the document head or body and returns the Collect instance as the result of resolved Promise. The script won't be loaded until `loadVGSCollect()` invoked. In order to speed up cross-domain loading, `dns-prefetch` and `preconnect` were added as a side effect.
 
 ```javascript 
 import { loadVGSCollect } from '@vgs/collect-js';
 
 // load script
-const VGSCollectInstance = await loadVGSCollect({
+const collect = await loadVGSCollect({
   vaultId: '<vault_id>', // required
   environment: '<environment>',
-  version: '2.0'
+  version: '2.2.1'
```

Comment:
version: <version_x.x.x> ?

console.log(Warning: pls use lock version in production, `<link to the lock version>`)